kickstart/disklayout_atomic.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: ptable
+name: Kickstart Atomic
+oses:
+- CentOS 7 Atomic
+- Fedora 22 Atomic
+- RedHat 7 Atomic
```

Comment:
Are these OS names going to work with foreman_templates?  Seems odd to have "Atomic" after the version number, not sure if it'll understand that.